### PR TITLE
Ct integration

### PR DESCRIPTION
**Pull request by Snugug**
Resolves the merge conflicts of #39

**Changelog**

```markdown
**New**

* Content Type Saving/Validation/Updating
* New Navigation

**Changed**

* Clean Up Routing
*  Clean Up Navigation
```

---
Resolves: #11 
Resolves https://github.com/punchcard-cms/punchcard/pull/40
Resolves https://github.com/punchcard-cms/punchcard/pull/36
Resolves #39

`DCO 1.1 Signed-off-by: Sam Richard <sam@snug.ug>`

### CODE REVIEW

File: lib/database/schemas.js [hunk 1]
```diff
@@ -26,10 +26,18 @@ const content = [
...
   {
+    name: 'sunrise-timezone',
+    type: 'string',
+  },
+  {
     name: 'sunset',
     type: 'dateTime',
   },
   {
+    name: 'sunset-timezone',
```
Comment (scottnath) [inline lib/database/schemas.js]: why can't there just be one timezone that both use?

Comment (Snugug) [inline lib/database/schemas.js]: Because someone may want to sunrise/set in different timezones

File: lib/routes/content.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+      }
+
+      // Validation
+      const validated = content.form.validate(req.body, type);
+
+      // utils.log(validated);
+
+      if (validated === true) {
+        // Sunrise/Sunset
+        const sunrise = utils.time.iso(req.body['sunrise-date'], req.body['sunrise-time'], 'EDT');
```
Comment (scottnath) [inline lib/routes/content.js]: `EDT` should be pulling from timezone selection

File: lib/routes/content.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        const sunset = utils.time.iso(req.body['sunset-date'], req.body['sunset-time'], 'EDT');
+
+        db(`content-type--${req.params.type.toLowerCase()}`).insert({
+          id: uuid.v4(),
+          language: 'us-en',
+          sunrise,
```
Comment (scottnath) [inline lib/routes/content.js]: `timezone`(s) should be here

File: lib/routes/content.js (hunk 2 above)
Comment (Snugug) [inline lib/routes/content.js]: Will need to do that, didn't want to spend cycles updating the universal input stuff to account for timezones. Let's capture as a story and we can update

**Pull request merged**